Chemdraw build fails on Big-endian: `cs_swapBytes.h: error: parse error in template argument list`

**Describe the bug**
Build fails on powerpc when compiling ChemDraw module.

**To Reproduce**
Run the build.

**Expected behavior**
Everything should compile. Unsupported components disabled at configure time.

**Logs**
```
/usr/bin/make  -f External/ChemDraw/CMakeFiles/ChemDraw.dir/build.make External/ChemDraw/CMakeFiles/ChemDraw.dir/build
make[2]: Entering directory `/opt/local/var/macports/build/rdkit-221e9e02/work/build'
[ 24%] Building CXX object External/ChemDraw/CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/Base64Converter.cpp.o
cd /opt/local/var/macports/build/rdkit-221e9e02/work/build/External/ChemDraw && /opt/local/bin/g++-mp-14 -DBOOST_IOSTREAMS_DYN_LINK -DBOOST_IOSTREAMS_NO_LIB -DBOOST_RANDOM_DYN_LINK -DBOOST_RANDOM_NO_LIB -DBOOST_REGEX_DYN_LINK -DBOOST_REGEX_NO_LIB -DBOOST_SERIALIZATION_DYN_LINK -DBOOST_SERIALIZATION_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_SYSTEM_NO_LIB -DCHEMDRAW_BUILD -DChemDraw_EXPORTS -DPLATFORM_BIGENDIAN -DRDKIT_DYN_LINK -DRDK_32BIT_BUILD -DRDK_BUILD_COORDGEN_SUPPORT -DRDK_BUILD_DESCRIPTORS3D -DRDK_BUILD_MAEPARSER_SUPPORT -DRDK_BUILD_THREADSAFE_SSS -DRDK_BUILD_YAEHMOP_SUPPORT -DRDK_HAS_EIGEN3 -DRDK_TEST_MULTITHREADED -DRDK_USE_BOOST_IOSTREAMS -DRDK_USE_BOOST_SERIALIZATION -DRDK_USE_STRICT_ROTOR_DEFINITION -DRDK_USE_URF -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/RingFamilies/RingDecomposerLib/src/RingDecomposerLib -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/expatpp/expatpp-code-r6-trunk/src_pp -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/expatpp/expatpp-code-r6-trunk/expat/lib -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/Code -isystem /opt/local/include/eigen3 -isystem /opt/local/libexec/boost/1.88/include -pipe -I/opt/local/libexec/boost/1.88/include -Os -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/libexec/boost/1.88/include -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -Wno-deprecated -Wno-unused-function -fno-strict-aliasing -Wall -Wextra -fPIC -std=gnu++20 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -DTARGET_API_LIB -D__linux -DHAVE_EXPAT_CONFIG_H -w -Wno-unknown-pragmas -Wno-error -MD -MT External/ChemDraw/CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/Base64Converter.cpp.o -MF CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/Base64Converter.cpp.o.d -o CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/Base64Converter.cpp.o -c /opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/Base64Converter.cpp
[ 25%] Building CXX object External/ChemDraw/CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/CDXAnnotation.cpp.o
cd /opt/local/var/macports/build/rdkit-221e9e02/work/build/External/ChemDraw && /opt/local/bin/g++-mp-14 -DBOOST_IOSTREAMS_DYN_LINK -DBOOST_IOSTREAMS_NO_LIB -DBOOST_RANDOM_DYN_LINK -DBOOST_RANDOM_NO_LIB -DBOOST_REGEX_DYN_LINK -DBOOST_REGEX_NO_LIB -DBOOST_SERIALIZATION_DYN_LINK -DBOOST_SERIALIZATION_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_SYSTEM_NO_LIB -DCHEMDRAW_BUILD -DChemDraw_EXPORTS -DPLATFORM_BIGENDIAN -DRDKIT_DYN_LINK -DRDK_32BIT_BUILD -DRDK_BUILD_COORDGEN_SUPPORT -DRDK_BUILD_DESCRIPTORS3D -DRDK_BUILD_MAEPARSER_SUPPORT -DRDK_BUILD_THREADSAFE_SSS -DRDK_BUILD_YAEHMOP_SUPPORT -DRDK_HAS_EIGEN3 -DRDK_TEST_MULTITHREADED -DRDK_USE_BOOST_IOSTREAMS -DRDK_USE_BOOST_SERIALIZATION -DRDK_USE_STRICT_ROTOR_DEFINITION -DRDK_USE_URF -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/RingFamilies/RingDecomposerLib/src/RingDecomposerLib -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/expatpp/expatpp-code-r6-trunk/src_pp -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/expatpp/expatpp-code-r6-trunk/expat/lib -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw -I/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/Code -isystem /opt/local/include/eigen3 -isystem /opt/local/libexec/boost/1.88/include -pipe -I/opt/local/libexec/boost/1.88/include -Os -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/libexec/boost/1.88/include -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -Wno-deprecated -Wno-unused-function -fno-strict-aliasing -Wall -Wextra -fPIC -std=gnu++20 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -DTARGET_API_LIB -D__linux -DHAVE_EXPAT_CONFIG_H -w -Wno-unknown-pragmas -Wno-error -MD -MT External/ChemDraw/CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/CDXAnnotation.cpp.o -MF CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/CDXAnnotation.cpp.o.d -o CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/CDXAnnotation.cpp.o -c /opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/CDXAnnotation.cpp
In file included from /opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/CDXObject.h:49,
                 from /opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/CDXStdObjects.h:46,
                 from /opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/CDXAnnotation.cpp:37:
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h: In function 'double SwapBytes(double)':
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:110:10: error: parse error in template argument list
  110 |   return SwapBytes<double,sizeof(double))>(m);
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:110:10: error: cannot resolve overloaded function 'SwapBytes' based on conversion to type 'double'
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:110:41: error: expected ';' before ')' token
  110 |   return SwapBytes<double,sizeof(double))>(m);
      |                                         ^
      |                                         ;
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:110:41: error: expected primary-expression before ')' token
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h: In function 'float SwapBytes(float)':
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:114:20: error: 'foud' was not declared in this scope
  114 |   return SwapBytes<foud,sizeof(float))>(m);
      |                    ^~~~
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:114:10: error: parse error in template argument list
  114 |   return SwapBytes<foud,sizeof(float))>(m);
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:114:10: error: cannot resolve overloaded function 'SwapBytes' based on conversion to type 'float'
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:114:38: error: expected ';' before ')' token
  114 |   return SwapBytes<foud,sizeof(float))>(m);
      |                                      ^
      |                                      ;
/opt/local/var/macports/build/rdkit-221e9e02/work/Release_2025_09_5/External/ChemDraw/chemdraw/chemdraw/cs_swapBytes.h:114:38: error: expected primary-expression before ')' token
make[2]: *** [External/ChemDraw/CMakeFiles/ChemDraw.dir/chemdraw/chemdraw/CDXAnnotation.cpp.o] Error 1
make[2]: Leaving directory `/opt/local/var/macports/build/rdkit-221e9e02/work/build'
make[1]: *** [External/ChemDraw/CMakeFiles/ChemDraw.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Configuration (please complete the following information):**
 - RDKit version: 2025.09.5
 - OS: macOS 10.6.8
 - gcc 14.3.0
 - Python version (if relevant): 3.13.11
 - Are you using conda? No

**Additional context**
MacPorts

--- a/External/ChemDraw/CMakeLists.txt
+++ b/External/ChemDraw/CMakeLists.txt
@@ -10,8 +10,8 @@ include(CMakePrintHelpers)
 # and will need to be fixed in the future
 if(RDK_BUILD_CHEMDRAW_SUPPORT)
   if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/chemdraw/chemdraw/CDXIO.h")
-    set(RELEASE_NO "1.0.11")
-    set(MD5 "8f2a65313eac027258423bd46feda5bf")
+    set(RELEASE_NO "1.0.12")
+    set(MD5 "8abbd599d798e980b63394a939a08b38")
     downloadAndCheckMD5("https://codeload.github.com/Glysade/chemdraw/tar.gz/refs/tags/v${RELEASE_NO}"
       "${CMAKE_CURRENT_SOURCE_DIR}/chemdraw-v${RELEASE_NO}.tar.gz" ${MD5})
 

